client/blocks/inline-help/popover.jsx

```diff
@@ -205,12 +211,23 @@ class InlineHelpPopover extends Component {
 		);
 	};
 
+	checkForClassicEditorOnAtomic( siteId, sitePlugins ) {
+		const classicPlugin = find( sitePlugins, { slug: 'classic-editor' } );
+		if ( ! classicPlugin.isActive ) {
```

Comment:
I ran into a few errors when testing this. 

Using debugger showed `sitePlugins` was empty and so as a result `classicPlugin` was undefined when the attempt to access `isActive` is made.

I tried this with two different atomic sites. Same result initially. Confirmed them both as atomic in SA. Both also had the classic editor plugin installed and listed as inactive.

Despite being atomic in SA and the classic editor plugin showing in WP admin, I found that one site would allow me to "install" the classic editor plugin. After doing so and deactivating the classic editor plugin, re-testing actually worked as expected.

The other atomic site still fails.

<img width="1798" alt="Screen Shot 2020-05-07 at 8 16 26 pm" src="https://user-images.githubusercontent.com/60436221/81288663-6e709980-90a8-11ea-8a8f-b53df0979bf2.png">


I hope this makes some sense. I'll try and narrow down the issue and update accordingly.

Reply:
If I add a check that classicPlugin exists it avoids the error initially. Undoing that change and deactivating the classic editor plugin in WP admin before re-testing actually allows the process to complete without errors.

Reply:
Ah, thanks -  I had the same thing, the plugins list was empty the first time I checked it, so I initially just activated it without checking it first, and then the plugin list was there, I thought I had just missed refreshing the browser after a change the first time, but it looks like the plugin list isn't populated until you activate one by default ... I think there is another action that gets the plugin list, will try and add this to the component mount and see if that resolves this.

Reply:
@aaronrobertshaw I pushed a changed which hopefully fixes the issue you had, can you give it a quick try and see if it does please? don't worry about testing it too much though as I still need to add in a bit of error handling.

Reply:
@glendaviesnz this is fixed the issue for me. Once the error handling is good to go I can create a fresh atomic site and give it another full test through if needed.